cairo/src/gas.cairo

```diff
@@ -177,7 +177,7 @@ namespace Gas {
         return expansion;
 
         no_expansion:
-        let range_check_ptr = [fp - 8];
+        let range_check_ptr = [fp - 9];
```

Comment:
Why ?
There is no local variable assignation before this line, there are 5 arguments (words_len, offset_1, offset_2, size_1, size_2) and one implicit that we want to retrieve range_check_ptr
By taking into account that a call instruction push next pc and current fp that are at fp-1 and fp-2, from my understanding fp - 8 is ok
What do I miss here ?